beacon-chain/chaintest/backend/simulated_backend.go

```diff
@@ -0,0 +1,82 @@
+// Package backend contains utilities for simulating an entire
+// ETH 2.0 beacon chain in-memory for e2e tests and benchmarking
```

Comment:
Is this true? The database writes to disk so its not fully in-memory. Was that the intention?